bosh-setup/azuredeploy.json

```diff
@@ -89,19 +66,42 @@
     },
     "enableDNSOnDevbox": {
       "type": "bool",
+      "defaultValue": true,
       "allowedValues": [
         true,
         false
       ],
       "metadata": {
-        "description": "A default DNS will be setup in the devbox if it is true. Need a manual reboot to take effect."
+        "description": "A default DNS will be setup in the devbox if it is true."
+      }
+    },
+    "tenantID": {
```

Comment:
tenantID, clientID, clientSecret are for the service principal. We have the guide in detail for customers.